transfer/transfer-06-consumer-pull-http/README.md

```diff
@@ -0,0 +1,477 @@
+# Implement a simple "Consumer Pull" Http transfer flow
+
+The purpose of this example is to show a data exchange between 2 connectors, one representing the
+data provider and the other, the consumer. It's based on a consumer pull usecase that you can find
+more details
+on [Transfer data plane documentation](https://github.com/eclipse-edc/Connector/tree/main/extensions/control-plane/transfer/transfer-data-plane)
+For the sake of simplicity, the provider and the consumer
+will be on the same machine, but in a real world configuration, they will likely be on different
+machines. The final goal of this example is to present the steps through which the 2 connectors will
+have to pass so that the consumer can have access to the data, held by the provider.
+
+Those steps are the following:
+
+* Running the provider connector
+* Running the consumer connector
+* Running a Http server that will receive the Endpoint Data Reference on the consumer side, that
+  contains the url to be used to get the data.
+* Register data plane instance for provider connector
+* Register data plane instance for consumer connector
+* Create an Asset on the provider (The asset will be the data to be shared)
+* Create an access policy on the provider (The policy will define the access right to the data)
+* Create a contract definition on the provider
+
+At this step, the connector should be able to fetch the catalog from the provider and to see the
+contract offer generated from the resources that have been created.
+
+Once the catalog is available, to access the data, the consumer should follow the following steps:
+
+* Performing a contract negotiation with the provider
+* Performing a transfer
+    * The consumer will initiate a file transfer
+    * The provider will send an EndpointDataReference to the consumer
+* The consumer could reach the endpoint and access the data
+
+Also, in order to keep things organized, the code in this example has been separated into several
+Java modules:
+
+* `connector`: contains the configuration and build files for both the
```

Comment:
As the module of the consumer and the provider are pretty similar, it could be simple to just keep the module and rename it `http-pull-connector` . What do you think ?

Reply:
Sorry for the confusion, I didn't mean replacing the current `connector` module. I was talking about the existing `provider/consumer` directories, but didn't regard the fact that they only contain config files and thus aren't Gradle modules. Yes, then just renaming `connector` to `http-pull-connector` will do. 

Could you update the module list in the README, as right now it states that the `connector` module contains both build and config files (should be just build file)?